sys/net/gnrc/transport_layer/udp/gnrc_udp.c

```diff
@@ -34,6 +34,11 @@
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+/* this allows testing sock_udp on GNRC without having IPv6 enabled */
+#ifndef GNRC_NETTYPE_IPV6
+#  define GNRC_NETTYPE_IPV6 GNRC_NETTYPE_TEST
+#endif
+
```

Comment:
I have the feeling this also falls into the yolo hack category :/

Reply:
Having the yolo hack in a .c file instead of a .h file would at least reduce the impact it has.

Reply:
Definitely a hack. Now I come from the other side and say: This should be a define (probably via `CFLAGS`) in the offending tests, as this is a test mock.

Reply:
Also, it prevents you (hopefully) from wacking all those moles that pop up now.